modules/core/client/controllers/menu.client.controller.js

```diff
@@ -38,7 +38,7 @@ function controllerSystemMenu($scope, $state, Authentication, Menus, $rootScope,
 
   $scope.$watch('project', function(newValue) {
     if (newValue) {
-      //console.log ('controllerSystemMenu.project = >' + newValue.code + '<');
+      //console.log ('controllerSystemMenu.project = >' + newValue.code + '<');// eslint-disable-line
```

Comment:
I am guessing this (and the files below) are not part of your commits?
Not sure how this happened, however when we have a REAL code review tool it should be easy to clean up the code review from unnecessary files before submitting it.

Reply:
They are --- was related to an issue generating a new contact for testing purposes. This likely has a tie in with the presence/absence of the "admin" role in user.roles

Reply:
Gotcha. Can we then remove the commented-out code if it is not being used?

Reply:
Gotcha. Can we then remove the commented code, since it is unnecessary?